app/view/templates/pages/help/pages/our-database.md

```diff
@@ -0,0 +1,20 @@
+## µGrowthDB
+
+µGrowthDB is a relational database written in MySQL. It contains 12 tables: project, study, experiment, biological replicate, biological replicate metadata, metabolite, metabolite per biological replicate, compartment, community, community member, taxa and perturbation. The definition of the tables that are most important to understand how the microbial growth database is structured are provided below.
```

Comment:
I may delete this page, because it just contains a bunch of implementation details that would not be useful to visitors (and are currently outdated). I might replace it with a "glossary" that explains terms used in the app.